manager/integration/tests/test_upgrade.py

```diff
@@ -243,6 +270,8 @@ def test_upgrade(longhorn_upgrade_type,
     rancher_secret_key = rancher_secret_key
     rancher_chart_install_version = rancher_chart_install_version
     longhorn_repo = longhorn_repo
+    flux_helm_chart_url = flux_helm_chart_url
+    flux_helm_chart_version = flux_helm_chart_version
```

Comment:
Why we need this kind of self assignment? it seems no any effect?

Reply:
Removed.